common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -274,8 +274,19 @@ private void clearRefQueue() {
         }
     }
 
+    /**
+     * Whether need to report leak.
+     * When the return value is true, {@link #reportTracedLeak} and {@link #reportUntracedLeak}
+     * can work normally, otherwise, they will have no effect.
```

Comment:
nit: `When the return value is {@code true}, {@link #reportTracedLeak} and {@link #reportUntracedLeak} will be called once a leak is detected, otherwise not.